Frontend/ui-driver/src/Flow.purs

```diff
@@ -1255,16 +1262,23 @@ bookingOptionsFlow :: FlowBT String Unit
 bookingOptionsFlow = do
   action <- UI.bookingOptions
   case action of
-    SELECT_CAB state -> do
+    SELECT_CAB state toggleDowngrade -> do
```

Comment:
can we rename the variable to downgrade instead of toggle downgrade

Reply:
this action is for downgrade only, and in case of toggle we are downgrading to all at once instead of check boxes so this variable is named as per behaviour